src/main/java/org/broadinstitute/hellbender/tools/sv/SVCallRecord.java

```diff
@@ -215,7 +222,9 @@ private static Pair<Boolean, Boolean> inferStrands(final StructuralVariantType t
             return Pair.of(Boolean.FALSE, Boolean.TRUE);
         } else {
             if (type.equals(StructuralVariantType.INV)) {
-            Utils.validateArg(inputStrandA != null && inputStrandB != null, "Cannot create variant of type " + type + " with null strands");
+                if (inputStrandA == null && inputStrandB == null) {
+                    return Pair.of(Boolean.TRUE, Boolean.TRUE);
```

Comment:
Thanks for catching this. I did some digging and I actually found our cleaned vcfs don't have STRANDS for INV types and I think this is fine. We do keep them there in intermediate vcfs since manta reports them that way, but we only retain ones with all 4 breakends (see [manta manual](https://github.com/Illumina/manta/blob/master/docs/userGuide/README.md#inversions) about that).